framework/Renderers/Source/SVGRenderer.swift

```diff
@@ -132,4 +132,8 @@ public class SVGRenderer: Renderer{
8WTWYnlT4XwE7639RI9pwKonM5HO1bYVkTzzVYV4BgRWGslonzsaSeekblngRHWdkPZQvmswNIgN3z0q9Q2b7V4tiYPyIpHn1wpuqkkV/dxyPUxjFny3Fon1q3lc+z8ibxAFuD8TUBvBqwHsh1I3CDDGNiOUc8QfSPcVW64l4lsey+PE//D+E22QduMztVzu2dteX6NambTNDzCwojeRo46akC6/HTeEeTHprf1Ev0FtB2VyTo9CEcZYxlD0WdrLOuY6nb/NqDWi/gd/5+gbeVeAn5vbTeULYTPu4CXmdXNFDixes223ly47Yxty0fSK2zHzkoI0c0lZyEKQ3atah/OrKOmad9ulL5aw1q3Q9wfjEexKWj2Rxa7JxsijJbxWqZ34LUGbMbrcPMvdFoUXnwjM3PvPmClbdDysX+c1DBefUXexFssjOgOvP2Ob6sHk1FyYNtBDzYyqx/b3tZD9HcxV2RqttS+x9if0PB9+Ww4/uyv1Ckeg1QU/iKyqYgLOnGEsvl9rmoteOrgA/FYf4hZ3UwB/ir1KMYHjH2IKrk07JGim0vO0iaDcGaz6GNRKPvrehdJmfqjDzH9sPg8KMNoGa9VdNlqxhEcYewj5O2QuT64yTZoHnyM9B6HtlegUlxK3OZ4rFM40fGgyE2CWhBeliuf2Y9tb+shejmmzRS5avHvVv1w2Rp4xyh4FG1F0FylaitXgkRln6IdvktC2/w+/RzrEeyGVxklc9egOWPdtM1vxjohurnkLDkXsLBms+hrsUFlBwHH/YP+QCWWijBaxmsVXa2d9wP9WPIn2mxxq2jVJmhxUCEumicczDmBgyys6J+KHJ/95Lw+EE7+3+zHtrcR09Qk+mrlM+vkI1obtfAUS/i6+jej4AK8ID9T1VbuJqDaKnAoW5Do2QsH1oRww6v0l+Y2eJ0xJ6aqasOE6OaSXN2nQzRgmE2iF7hxSmVrvOiiPXHF4ROVmiTCaBmvRfQtKlEETFEp3/H999SijW3Q4lCrItEF8TlsgygdVvQVWsR8yp9xWtD92Pc2YqrkmqqoULNqh0L5FfysH990uAO+lM/o5Z6yGJ8D9soH/V8RnBACh7L5RecvWubMl6pwZIY0WNzwUavZdwMS2EzIpUSOs4W15CzEZ4ZoQDebRU8DtM01W4ePtBQ3ajfBn/MwWsdrEb1CS2WLy0jXqdPFquBs26Al6kvNAWDNe0jJsRG9RhPdFSB6oB/73kbMQwGH33x8IT8P7Tnp4c00fJbVmO5adRoH+HqX55cAOwMerxUCh7KZROcc96BZX5GwuOGjZoboPQ3Rv+RdtZScBS8L0YBuDiv6ZHylpcoA7WR0hofROl6L6Gu1lBcD2YrufNPcaOPZUc9L0cMH7YTc9HJW8FUrISB0+DpIdH34AaKb/Nj3NmJuxGY9mePGWT3tavlNpdiuJOLLoCoFVXjTp1gmt3J8tb17k0XgUDar6Pzkh3Py/tHsxiQ6K8RolftCvJ/mkoa6Ycwm8WLj9Om9utK4OJrjxlaVulZMmJbxWkRfrxKHxNvzGjxT5Q7rW+j3AaGDdl6YBPybfuHiRTf5se9txHyAbP3HkNEA3zMU7UpPk9nvEMXYOX1buL20W7JWbiBQpiX7OeQu6wzwmdFi5jEpcCibVXQ2GzgqPs1uzKIP1bdfSUJ0c0lD3TBms3gVektNAgK+GKkqkSvCaBmvdSMnd87sVcT7MgvxnTLnY6Ft0BbBqX54uA+1m/qVKf6OzWVl1RGIbvZj39uI+dqN0+qGtCwf63k3CrK0qe975IvNB+RFhqsbGulVygPu2h5HFX8NXWOA9G3K8sQPatMWyhYketE6FL7BrG7Mot+M7AMi8w95ZDOXNNQNY9Y5hkVMHtnkrzuuadht/MBxFA07ai9CedB4VT2/6KvEZ/XdOM8KgF7S+gwwzDZofQvV8WPHMfzTaCvMmh4sutmPfW8j5xZg3csr77+hhK9IA4XhHjhHzGHV+6PE3MEHWLnMx5LPw61fdaTF4YxRm0/QE/hHp0ZA1b7OrVaezeXTVrTcxYayWUUXSuZmWN2YRXdVoO31zDXVI0U3lzTUDWPWeRLTlvQWh6bJWzex5W/B7T/R+fKR1JLVnJoOcWliGa+q5xcd1/ZnFzYjq7dYsErXMNbzo0p582cXtK+A4R3Z8hvRsO/Fi27xY9vbyCl4W19ivOoYkyPuj6r4xvCcOI79azEQL34M7qVX+ApxM4zasdPlVo4VNY7Xm1n/ubaWh7JZRedLFj60ujGLzvaeAJoWIk8e2cwl/eqGMWvwAyQWM/bGbj5KHq/sRwOePVHFj1uVqO2Gk0Hj1erpog8ul5ee2a14bhBXoTTFgSP/Fvs7u6BlDueppvzxp36nkYtu9mPb24thyaqK2vjEcSPf11+D53KrHLWnJ6gZsGDn+BhHaUJXo/hg4xdCAX9lWstE8tkfiis9zZpPLfA/DGWziH6oGIXLLW4sorOe3ydl5y08PFIdNAJLBqgbxqzwbdjiTRKfvcZ4sgZ/Y77ISEtPcRbvOzxb3ARZxqvX00RPqnlzonNymwsyu+Z8raN2yMHM43Bf+JGgNVmd553XoizAZ+SiW/zY9ZaIFGNxH4L/XOq+XG29vWz578fqdvGw96Iutai3VzK74NxZxGqaLED3mkvdl6utt5ctBeP5PsjDt2DH5te/MertFcKmld+mePNyz/gudUeuwt4SBEEQBEEQBEEQBEEQBEEQBEEQBEEQBEEQBEEQBEEQBEEQBEEQBEEQBEEQBEEQBEEQBEEQBEEQBEEQBEEQBEEQBEEQBEEQBEEQBEEQBEEQBEEQBEEQBEEQBEEQBEEQBHHl8j8u6OKnh8GDMAAAACV0RVh0ZGF0ZTpjcmVhdGUAMjAxOS0wNS0yM1QyMDowMTo0MyswMjowMAZrx/8AAAAldEVYdGRhdGU6bW9kaWZ5ADIwMTktMDUtMjNUMjA6MDE6NDMrMDI6MDB3Nn9DAAAAAElFTkSuQmCC"
```

Comment:
This string displays a message "Use AGG Renderer to get base64 image"

Reply:
This is a good thing to include as a comment in the code, so that future code readers know what this string is :)

(This is probably irrelevant, because of my above suggestion to delete this method.)